package.json

```diff
@@ -23,96 +23,97 @@
     "icon": "images/hubspot-logo.png",
     "main": "./main.js",
     "activationEvents": [
-        "onLanguage:hubl"
+        "onLanguage:html-hubl",
+        "onLanguage:css-hubl"
```

Comment:
Should we move the indention rules into the language config?

Reply:
Good question. Previously that did not work but I'll try that again now that we're not hijacking existing language types.

Reply:
Thanks for pointing that out, that worked. 

I also removed `main.js` the `activationEvents` for now. The docs suggest that you do need `activationEvents` but the extension does not need to work properly (tested, works for hubl types and not original langs). Also, their example language configuration project does not use them: https://github.com/microsoft/vscode-extension-samples/blob/master/language-configuration-sample/package.json